filebeat/autodiscover/builder/hints/logs.go

```diff
@@ -25,7 +25,7 @@ const (
 )
 
 // validModuleNames to sanitize user input
-var validModuleNames = regexp.MustCompile("[^a-zA-Z0-9]+")
```

Comment:
change lgtm, I think we should probably make it a type and when we cast it it should fail? wdyt?

Reply:
I don't know too much about this part of the code, w
here is this module name configured in the yml?

Reply:
It comes from autodiscover hints, so normally user would enter it through docker/k8s labels

Reply:
Now I realize I'm not sure how the change would look :innocent: , let's sync tomorrow